File: Greedy, BruteForce/Kotlin/1476 날짜 계산.kt
import java.util.*
import kotlin.math.*

fun main(args: Array<String>) {
    var sc = Scanner(System.`in`)
    var e = sc.nextInt(); var s = sc.nextInt(); var m = sc.nextInt(); var answer = 0;
    var se = 0; var ss = 0; var sm = 0;
    while(true) {
        se++; ss++; sm++; answer++;
        if(se > 15) se = 1; if(ss > 28) ss = 1; if(sm > 19) sm = 1;
        if(se == e && ss == s && sm == m) break
    }
    println(answer)
}

File: Greedy, BruteForce/Python/1476 날짜 계산.py
e,s,m = map(int, input().split()); 
answer = 0; se = 0; ss = 0; sm = 0
while True :
    answer += 1; se += 1; ss += 1; sm += 1
    if se > 15 : se = 1
    if ss > 28 : ss = 1
    if sm > 19 : sm = 1
    if e == se and s == ss and m == sm : break
print(answer)
